src/helpers/augmentedReality/ARSupportingDevice.js

```diff
@@ -0,0 +1,11 @@
+import { osVersion } from 'expo-device';
+import { useContext } from 'react';
+
+import { device } from '../../config';
+import { SettingsContext } from '../../SettingsProvider';
+
+export const ARSupportingDevice = () => {
+  const { globalSettings } = useContext(SettingsContext);
+
+  return { isARSupported: osVersion >= globalSettings?.augmentedReality[device?.platform] };
```

Comment:
how did you found out which device is supported by viro technically to compare here?

Reply:
I think it is better with this document.
Doc: [isARSupportedOnDevice](https://viro-community.readme.io/docs/isarsupportedondevice)

https://github.com/ikuseiGmbH/smart-village-app-app/pull/593/commits/735aa35dda1b5631991b68760609d245dd566f5e